package.json

```diff
@@ -7,6 +7,7 @@
   "scripts": {
     "test-typings": "tsc --lib ES2015 --noEmit typings/*.ts",
     "test": "mocha index.spec.js lib/**/*.spec.js && npm run test-typings",
+    "posttest": "npm run lint",
```

Comment:
any reason for not running it as part of the test script above?

Reply:
I don't have any strong opinions not to. Which ever you prefer honestly 😄. Most patterns I've seen is handling it in the `posttest` cycle vs. in the main `test` cycle` We could do this `pretest` as well to fail early if you choose